ccf/Docker Test Algorithm/Data/invert.py

```diff
@@ -0,0 +1,20 @@
+import sys
+
+import PIL.ImageOps
+from PIL import Image
+
+
+def process(name):
+    print(name)
```

Comment:
Is this a testing print statement?

Reply:
It was not 100% necessary so i removed it in 1e32d9b.